recipes/seqan_tcoffee/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: seqan_tcoffee
+  version: '1.13.3'
+
+source:
+  fn: seqan-src-2.1.1.tar.gz
+  url: http://packages.seqan.de/seqan-src/seqan-src-2.1.1.tar.gz
+  sha256: df083d5939d7e73fd00bbc6bb291ac7ea7b8d446cf89a9758fdffdc0eca9ec1c
+
+build:
+  number: 1
```

Comment:
This one should start with build number 0.

Reply:
Yes, I see you have already changed it :+1: